voice/methods/conferences/getConferences.md

```diff
@@ -0,0 +1,123 @@
+{% method %}
+
+## Retrieve Account's Conferences
+Returns a max of 1000 conferences, sorted by createdTime from oldest to newest
+
+### Request URL
+
+<code class="get">GET</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/conferences`
+
+#### Basic Authentication
+
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
+
+---
+
+### Supported Parameters
+
+| Parameter      | Description                                                                                                                      | Mandatory |
+|:---------------|:---------------------------------------------------------------------------------------------------------------------------------------------|
+| pageSize       | Specifies the max number of conferences that will be returned. Range: integer numbers from 1 to 1000. Default value is 1000.     | No        |
```

Comment:
What you have is certainly fine, but looks like in the docs the range specifications are pretty consistent. 
```suggestion
| pageSize       | Specifies the max number of conferences that will be returned. Range: integer values between 1 - 1000. Default value is 1000.     | No        |
```